LICENSE.md

```diff
@@ -67,58 +67,59 @@ The User may not limit access to the Mod in any way, except as provided by the G
 The User may decompile compiled binaries of the Mod.  
 The User may examine the Mod's source code.
 
-5. Right to distribute
+5. No right to distribute
```

Comment:
Why change this? The sections are for descriptive purpose, not defining purpose. This section covers the right to distribute the mod...yes, it's not allowed without permission from the author, but that's not the purpose of the section headings.